ktor-io/common/src/io/ktor/utils/io/core/BufferPrimitives.kt

```diff
@@ -24,9 +24,7 @@ public inline fun Buffer.forEach(block: (Byte) -> Unit) {
  */
 public fun Buffer.readUByte(): UByte = readByte().toUByte()
 
-@Suppress("EXTENSION_SHADOWED_BY_MEMBER", "DEPRECATION")
-@Deprecated("IoBuffer is deprecated. Use Memory or Input instead.")
-public inline fun IoBuffer.readUByte(): UByte = (this as Buffer).readUByte()
+public inline fun ChunkBuffer.readUByte(): UByte = (this as Buffer).readUByte()
```

Comment:
ChunkBuffer extends Buffer, are those functions needed for ChunkBuffer?

Reply:
We will drop members to work with primitive types and leave extensions only